llmfoundry/models/layers/ffn.py

```diff
@@ -32,9 +42,9 @@ def _resolve_ffn_hidden_and_exp_ratio(
         ffn_hidden_size = int(d_model * expansion_ratio)
         if ffn_hidden_size != d_model * expansion_ratio:
             raise ValueError(
-                f'`d_model * expansion_ratio` ({ffn_hidden_size}) must be an integer.'
+                f'`d_model * expansion_ratio` (={d_model * expansion_ratio}) must be an integer.'
```

Comment:
nit: include d_model and expansion_ratio values in the error message

Reply:
Thank you!

Reply:
eg: 
```
ValueError: `d_model * expansion_ratio` must be an integer (d_model=128; expansion_ratio=1.231; d_model * expansion_ratio=157.568).
```